proposals/0672-tuist-react-native

```diff
@@ -0,0 +1,85 @@
+---
+title: Tuist.io in React Native
+author:
+- Riccardo Cipolleschi
+date: 2023-06-19
+---
+
+# RFC0672: Tuist.io in React Native 
```

Comment:
nit: Ideally this RFC should be called "Using a declarative approach to describe iOS project files", and explains why it's important, also mentioning some of the practical problems we had in the past.

Tuist is then one of the solution and mention that it's the preferred.

It seems here you focus too much on the tool and don't explain why we need it in the first place (it might be obvious for a iOS engineer, but might be not for a Android/Web one).